modules/dhcp/subnet.rb

```diff
@@ -17,11 +17,19 @@ class Subnet
     include Proxy::Log
     include Proxy::Validations
 
-    def initialize network, netmask
-      @network   = validate_ip network
-      @netmask   = validate_ip netmask
-      @options   = {}
-      @timestamp = Time.now
+    def initialize network, netmask, options = {}
+      @network       = validate_ip network
```

Comment:
nitpick: you can get rid of assignment formatting -- simple spaces around '=' would suffice.